spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java

```diff
@@ -65,32 +62,14 @@ void testGetLoggingOptions() {
         .isNotNull()
         .contains("spring-cloud-gcp-logging")
         .contains("Spring");
-
-    loggingAppender.start();
-    // java-logging-logback appender default is ERROR
-    verify(logging).setFlushSeverity(com.google.cloud.logging.Severity.ERROR);
-  }
-
-  @Test
-  void testDisablingFlushLevel() {
-    LoggingAppender loggingAppender = new TestLoggingAppender();
-    loggingAppender.setLogDestinationProjectId("my-log-destination-project");
-    loggingAppender.setFlushLevel(Level.OFF);
-
-    loggingAppender.start();
-    verify(logging, never()).setFlushSeverity(any());
   }
 
   @Test
   void testSetLogDestinationProjectId() {
     LoggingAppender loggingAppender = new TestLoggingAppender();
     loggingAppender.setCredentialsFile("src/test/resources/fake-project-key.json");
     loggingAppender.setLogDestinationProjectId("my-log-destination-project");
-    assertThat(loggingAppender.getLoggingOptions().getCredentials()).isNotNull();
-    assertThat(loggingAppender.getLoggingOptions().getProjectId()).isEqualTo("my-log-destination-project");
-    assertThat(loggingAppender.getLoggingOptions().getUserAgent())
```

Comment:
This seems to be valid check for [this line](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/72ee57fb075a2866ba6e80cf3312f2bf911e8b07/spring-cloud-gcp-logging/src/main/java/com/google/cloud/spring/logging/LoggingAppender.java#L43), if we setup this test with `LoggingAppender loggingAppender = new LoggingAppender();` instead

Reply:
OK, I restored the original assertions.